engines/sci/resource.cpp

```diff
@@ -854,7 +861,8 @@ void DirectoryResourceSource::scanSource(ResourceManager *resMan) {
 }
 
 void ExtMapResourceSource::scanSource(ResourceManager *resMan) {
-	if (resMan->_mapVersion < kResVersionSci1Late) {
+	if (resMan->_mapVersion < kResVersionSci1Late &&
+	    !(getLocationName() == "message.map" && g_sci->getLanguage() == Common::KO_KOR)) {
```

Comment:
Since this condition gets repeated 4 times, and it's for Korean only, it would be better if it was put in a separate function, e.g. isKoreanMap()